src/client/components/detail-chart/detail-chart.component.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './detail-chart.style.css';
+import PropTypes from 'prop-types';
+
+import {
+  ResponsiveContainer,
+  LineChart,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Brush,
+  Line,
+  ReferenceLine,
+} from 'recharts';
+
+export default function DetailChart({ data }) {
+  return (
+    <div className="App">
+      <ResponsiveContainer
+        className="detail-chart-container"
+        width="100%"
+        height={400}
+      >
+        <LineChart data={data} margin={{ right: 80 }}>
+          <XAxis
+            stroke="#DDE0E3"
+            dataKey="timestamp"
+            height={50}
+            domain={['dataMin', 'dataMax']}
+            tick={{
+              transform: 'translate(-20, 0)',
+              fontFamily: 'Roboto',
+              textAlign: 'right',
+              fontSize: 10,
+              fill: '#A3A3A3',
+            }}
+          />
+          <YAxis
+            stroke="#DDE0E3"
+            type="number"
+            domain={['dataMin', 'dataMax']}
+            tick={{
+              transform: 'translate(-20, 0)',
+              fontFamily: 'Open Sans',
+              fontSize: 10,
+              fill: '#666666',
+            }}
+            fill="none"
+            ticks={[10, 15, 20, 25, 30]}
+          />
+          <Tooltip />
+          <ReferenceLine y={20} stroke="#6F9C67" strokeWidth={2} />
+          <ReferenceLine y={15} stroke="#9C5256" strokeWidth={2} />
+          <ReferenceLine y={25} stroke="#9C5256" strokeWidth={2} />
```

Comment:
@MirSzabo As we talked about let's make these props and then we are good to go! I will Approve it and you can add this fix 👍

Reply:
I added props exactly as we discussed during online class (also in more lines)